### ISSUE TITLE
VS libs aren't currently working

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/apothecary#241]
This is because we haven't been  using the apothecary libs in the nightly builds but the old ones from the ci server. 
So didn't realize they were failing. 

Two things currently seem to be causing errors 
 - [x] svgtiny wasn't being built so  it wasn't being included in the libs package #240 
 - [x] assimp uses pugixml and it is clashing at link time with the pugixml we use in ofXml - that's a tough one to solve - might involve the`/FORCE:MULTIPLE` linker flag to allow both symbols to exist 😬

so might take a day or two to untangle 🙂

**Comment by dimitre** [comment]
Can't we bump pugixml to 1.12.1 in OF, so it uses the same Assimp version?

**Comment by ofTheo** [comment]
we could yeah - but it doesn't get rid of the fact the symbols are linked twice. 
I do have a fix for that though. 

But I'm still this linker issue with the allAddonsExample  ( emptyExample works fine  ) using the latest libs from apothecary slowing trying to narrow it down. 

```
LINK : fatal error C1900: Il mismatch between 'P1' version '20210202' and 'P2' version '20180423' [D:\a\openFrameworks\openFrameworks\examples\templates\allAddonsExample\allAddonsExample.vcxproj]
[1119](https://github.com/ofTheo/openFrameworks/actions/runs/3198652746/jobs/5223450037#step:6:1120)
```

I thought it was something to do with the windows 8.1 sdk being used in some formulas but it might be something completely different

**Comment by ofTheo** [comment]
Thanks @danzeeeman 
Some of the libs were still being built with v141 while most were being built with v142 

So just had to track down which ones through tedious trial and error ( and with no help from visual studio except that message 🙃 ). 

All good now, but we'll probably want a big rethink at some point.

**Comment by ofTheo** [comment]
closed by #242
